### PR TITLE
Fixed assign documents modal.  Closes #148

### LINKED ISSUE #148: [Bug] Assigning documents in offboarding is broken
A modal is supposed to come up when you click on assign documents button in offboarding.

### PR DESCRIPTION
**Pull request by jscottchapman**

### CODE REVIEW

File: app/controllers/account/employee/offboard/options.js [hunk 1]
```diff
@@ -6,21 +6,5 @@ import addEdit from 'granite/mixins/controller-abstractions/add-edit';
...
   states,
-  stateIsMontana: computed.equal('model.addressState', 'MT'),
-
-  init () {
-    this._super(...arguments);
-
-    run.scheduleOnce('afterRender', () => {
-      let model = this.get('model');
-
-      model.setProperties({
```
Comment (James1x0) [inline app/controllers/account/employee/offboard/options.js]: I think we need these defaults set. Can you move this to the routes' setupcontroller hook?

File: app/templates/components/input/document-selection-modal.hbs [hunk 2]
```diff
@@ -38,7 +38,7 @@
       {{#each results as |document|}}
         {{list-item/document-option file=document remove=(contains document selectedDocuments) onRemove=(action "removeDocument") onAddition=(action "addDocument")}}
       {{else}}
-        {{#unless}}
+        {{#unless results}}
```
Comment (James1x0) [inline app/templates/components/input/document-selection-modal.hbs]: I don't think we need the "unless" here. #each's else block is effectively the same thing.

File: app/routes/account/employee/offboard/options.js [hunk 3]
```diff
@@ -1,3 +1,15 @@
 import Route from '@ember/routing/route';
 
-export default Route.extend({});
+export default Route.extend({
+
+  setupController(model, controller) {
```
Comment (James1x0) [inline app/routes/account/employee/offboard/options.js]: Argument order is backward, have you confirmed this sets defaults properly???

File: app/routes/account/employee/offboard/options.js [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+    this._super(controller, model)
```
Comment (James1x0) [inline app/routes/account/employee/offboard/options.js]: Use this._super(...arguments)

File: app/controllers/account/employee/offboard/options.js (hunk 1 above)
Comment (jscottchapman) [reply app/controllers/account/employee/offboard/options.js]: Yes

File: app/routes/account/employee/offboard/options.js (hunk 3 above)
Comment (jscottchapman) [reply app/routes/account/employee/offboard/options.js]: It does not.  I will put them in proper order.

File: app/routes/account/employee/offboard/options.js (hunk 4 above)
Comment (jscottchapman) [reply app/routes/account/employee/offboard/options.js]: Done.

**Pull request merged**